src/ButtonSelect/ButtonSelect.js

```diff
@@ -0,0 +1,46 @@
+
+import React, { useState, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Button';
+
+const menuStyles = {
+  width: '100%',
+  top: '100%'
+};
+
+const ButtonSelect = ({
+  text,
+  children,
+  ...Attr
+}) => {
+  const [toggled, setToggled] = useState(false);
+
+  return (
+    <Fragment>
+      <div className="more-menu__wrapper">
+        <Button onClick={() => setToggled(!toggled)} {...Attr}>
+          <i className="material-icons">{`keyboard_arrow_${toggled ? 'up' : 'down'}`}</i>
+          {text}
+        </Button>
+        
+        {toggled && (
+        <div className="more-menu open" style={menuStyles}>
```

Comment:
We shouldn't have inline styles.

Reply:
inline styles are temporarily for preview, the component template will be updated soon